src/ONGR/DemoBundle/Controller/ContentController.php

```diff
@@ -79,10 +79,14 @@ public function pageAction($slug)
         /** @var ContentService $service */
         $service = $this->get('ongr_content.content_service');
 
-        $document = $service->getDocumentBySlug($slug);
+        try {
+            $document = $service->getDocumentBySlug($slug);
 
-        if (!$document) {
-            throw $this->createNotFoundException('The content page does not exist');
+            if (null === $document) {
```

Comment:
@saimaz do you know in which situation ES would return null instead of exception, when Document is not found?